.jenkins

```diff
@@ -70,6 +70,7 @@ pipeline {
                         sh '''rm -rf build && mkdir -p build && cd build && \
                               cmake \
                                 -DCMAKE_BUILD_TYPE=Debug \
+                                -DCMAKE_CXX_CLANG_TIDY="/opt/rocm/llvm/bin/clang-tidy;-warnings-as-errors=*" \
```

Comment:
This won't work since it just uses the regular `clang++` as compiler and adds the compile flags passed by `Kokkos`. clang++ doesn't understand `--amdgpu-target=...` for example.
What I did was to look at how `hipcc` invokes `clamg++`, then use clang++ as compiler and add the extra flags manually. This also required to disable forcing to set the architecture since `clang++`, as already mentioned above doesn't understand the `--amdgpu-target=...` flag.